modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableRowSkinBase.java

```diff
@@ -349,12 +349,13 @@ protected ObjectProperty<Node> graphicProperty() {
                 height = h;
             }
 
-            width = tableCell.prefWidth(height);
-
             if (isVisible) {
                 if (fixedCellSizeEnabled && tableCell.getParent() == null) {
                     getChildren().add(tableCell);
                 }
+                // Note: We have to determine the pref width here because the add operation above may trigger the skin
+                // creation first, which is what makes it possible to get a correct value here in the first place.
```

Comment:
The comment needs some rewording. As is now, it only explains why the change proposed in this PR (moving the `prefWidth()` call, that is), but then it will be out of context.

Something like:
```
// prefWidth() has to be called only after the tableCell is added to the tableRow, if it wasn't already. 
// Otherwise it might not have its skin yet, and its width value would be 0.
```